battleship.py

```diff
@@ -219,8 +347,10 @@ def updateBoard(data, board, row, col, player):
 Returns: None
 '''
 def runGameTurn(data, row, col):
-    return
-
+    if data["boardcom"][row][col] ==  SHIP_CLICKED or EMPTY_CLICKED :
+        return None

```

Comment:
This is not the proper way of writing if condition with or.
Please write the correct condition.